packages/react-mutation-mapper/src/model/LollipopSpec.ts

```diff
@@ -8,12 +8,15 @@ export type LollipopSpec = {
     count: number;
     group?: string;
     placement?: LollipopPlacement;
-    label?: {
-        text: string;
-        textAnchor?: string;
-        fontSize?: number;
-        fontFamily?: string;
-    };
+    label: LollipopLabel;
     color?: string;
     tooltip?: JSX.Element;
 };
+
+export type LollipopLabel = {
+    text: string;
+    textAnchor?: string;
+    fontSize?: number;
+    fontFamily?: string;
+    show: boolean;
```

Comment:
I am not sure if `show` should be part of the model. I think it is `state` not `model`.